maple/pmore.c

```diff
@@ -193,23 +193,23 @@
 
 // Colors
 #define PMORE_COLOR_HEADER1 \
-    ANSI_COLOR(47;34)
+    ANSI_COLOR(47; 34)
 #define PMORE_COLOR_HEADER2 \
```

Comment:
I think this part should be recovered because of those defects discussed in comment ( #26 ).

Reply:
Resolved in d95ff15.